newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,56 @@
+* [Odra.dev releases their first documentation](https://twitter.com/odradev/status/1632711381257539585). (06.03.2023)
+
+### Conferences
+* [ETHDenver 2023](https://www.ethdenver.com/) - Ethereum event from February 24 to March 5, 2023. Videos are available [here](https://www.youtube.com/playlist?list=PLAy4HNUNlzRlPMWDNd1Oxun1X54EqP8K1).
+
+### Twitter
+* [Thread why ZK is the future of global finance](https://twitter.com/contangojosh/status/1631351358409302017). (02.03.2023)
+* [Thread about zkCPUs](https://twitter.com/cryptodavidw/status/1631416735327834115). (03.03.2023)
+* [Thread on Zero Knowledge Cryptocurrency](https://twitter.com/web3capo/status/1632329080161943552). (05.03.2023)
+* [Thread exploring what zero-knowledge proofs are](https://twitter.com/0x_Dxve/status/1632637164348661772). (06.03.2023)
+* [Thread with thoughts on ZK-EVMs](https://twitter.com/finestonematt/status/1632663969352523778). (06.03.2023)
+
+### YouTube Clips
+* [zk-SNARKs Explained - Binance’s Proof of Reserves System](https://www.youtube.com/watch?v=N5Qip04KIfo) - video sortly explaining zk-SNARKs. (01.03.2023)
+* [Zero-Knowledge Proof: Groth16](https://www.youtube.com/watch?v=VQyDSxB9Bls) - workshop by Alexey Kalmykov. (03.03.2023)
+* [STARKDenver Conference 2023 - Defining Onchain Possibilities](https://www.youtube.com/watch?v=0ugBSFxaJjA) - lecture from ETHDenver 2023. (03.03.2023)
+* [Polygon zkEVM mainnet](https://www.youtube.com/watch?v=CxWvaErjKsk) - lecture from ETHDenver 2023 by Jordi Baylina. (04.03.2023)
+* [How to build and deploy on-chain AI/ML with zero-knowledge machine learning](https://www.youtube.com/watch?v=YqnVAL3kkMk) - lecture from ETHDenver 2023 by Jason Morton. (04.03.2023)
+* [HCPP22 | Ariel Gabizon - Zero-knowledge proofs: From novice to master](https://www.youtube.com/watch?v=z8-U2oABnaA) - lecture from Hackers Congress Paralelni Polis by Ariel Gabizon. (05.03.2023)
+* [Delendum -  Daniel Kang, Scaling Zero Knowledge Proofs to ImageNet Scale Models](https://www.youtube.com/watch?v=am20GH2nEpM) - lecture by Daniel Kang. (07.03.2023)
+ 
+### Podcasts and other talks
+* Weekly [Zero Knowledge Podcast](https://zeroknowledge.fm/266-2/) episode - *Episode 266: ZK Hardware Sessions with Zprize Pt. 1.* (01.03.2023) 
+* Weekly [Devs Do Something](https://www.devsdosomething.fm/episodes/jordi-baylina-a-deep-dive-into-the-polygon-zkevm) episode - *A Deep Dive into the Polygon zkEVM with Jordi Baylina*. (02.03.2023)
+* [Arbitrum talk](https://twitter.com/arbitrum/status/1631353111397662720) - Arbitrum x Vela. (02.03.2023)
+* [Mina’s zkBridge to Ethereum — with =nil; Foundation](https://twitter.com/i/spaces/1YqJDoANeWDGV?s=20). (03.03.2023)
+* [Zero-Knowledge Cryptography in 2023: Privacy Becomes Practical](https://twitter.com/connect3world/status/1632383969215000578). (03.03.2023)
+* [Sovereign Rollups on Bitcoin](https://twitter.com/CelestiaOrg/status/1632892223540060160). (07.03.2023)
```

Comment:
I checked and wasn't able to find ZK in Celestia. I'd remove those two and any other mention of Celestia from AwesomeZK.